src/many.js

```diff
@@ -0,0 +1,33 @@
+// many.js
+export function setMany(entries, set) {
```

Comment:
I think the behavior implemented here is a more complicated API than is justified for such a simple library.

I think the only capabilities we should support would be:

```js
setMany([ ["prop1","val1"], ["prop2","val2"] ]);

setMany({ prop1: "val1", prop2: "val2" });
```

Of course, the contents of each value don't have to be strings (as I'm showing above), they can be rich objects/arrays. But we shouldn't reach into those values in anyway, just pass them along to `set()`, which handles its own serialization of the values being set.

Also, the contract of `set()` is that it returns `true` (or a `Promise<true>`) if the setting succeeded, or throws an exception if it failed to be set (like quota-exceeded). Your logic here ignores a rejected-promise that would come back from `set()` (as opposed to an exception being synchronously thrown).

The example I posted previously, with `Promise.all()`, would "catch" the rejected promise from any of the sets, and therefore propagate that failure outward to the calling code to detect.

SIDE NOTE: there is a "bug" of sorts in the current library implementations, in that `idb`, `opfs`, and `opfs-worker` are all async (as intended), but `cookie`, `local-storage`, and `session-storage` are all sync -- and they should instead be async. So need to fix that for consistency.